Dockerfile

```diff
@@ -3,7 +3,7 @@
 # SPDX-License-Identifier: Apache-2.0
 
 ############# golang-base
-FROM golang:1.16.3 AS golang-base
+FROM golang:1.16.15 AS golang-base
```

Comment:
Nit: you could also keep in sync the go version in https://github.com/gardener/terraformer/blob/72966630f66e43f8d8fd4c4ac16d8c12685a007d/.test-defs/pod-e2e.yaml#L18

Reply:
Thanks I missed that.